vtex-sites/casinofr.store
# Feature/ sign in button

## What's the purpose of this pull request?
It adds a new style to the SignInButton component

## How does it work?
Preview:
![image](https://user-images.githubusercontent.com/57547336/171857989-b3625078-4c83-49be-b43e-4748fefdad68.png)

Hover:
![image](https://user-images.githubusercontent.com/57547336/171858203-0f1fd56d-c4da-43d0-b250-284f61f3416f.png)

header overview:
![image](https://user-images.githubusercontent.com/57547336/171903235-da905016-b17a-4b00-a497-6235e87c8d02.png)

## How to test it?

If the user isn't logged in, It should redirect to the `/login` page
but if the user is logged, it should redirect to the `/account` page
